reference/orders.v2.oas2.yml

```diff
@@ -2897,10 +2905,12 @@ definitions:
       Note:
       - `xxx` and `xxx_customer` always hold the same value, updating either `xxx` or `xxx_customer` will change value for both of those fields.
       - If both fields `xxx` and `xxx_customer` are present, they must have same value.
-      - If field `xxx_merchant` is provided with empty value (`\"\"`) or null (`null`), it will fallback to have the value from `catalog`.
-      - If both fields `xxx` and `xxx_customer` are provided with empty value (`\"\"`) or null (`null`), those fields will fallback to have the value of `xxx_merchant`.
-      - In case of adding product, if `xxx_merchant` is omitted, it will fallback to have the value from `catalog`.
-      - In case of adding product, if both fields `xxx` and `xxx_customer` are omitted, or provided with empty value or null, those files will fallback to have the value of `xxx_merchant`.
+      - Empty strings `''` and `null` are invalid for `xxx`, `xxx_customer`, and `xxx_merchant`.
+      - When updating an existing order product without changing the variant, product, or product options, if `xxx_merchant` is not part of the request it will not be updated.
+      - When updating an existing order product, if both fields `xxx` and `xxx_customer`are not part of the request they will not be updated.
+      - When adding a product to an existing order or changing the variant, product, or product options.
+      - Empty strings `''` and `null` are invalid for `xxx`, `xxx_customer`, and `xxx_merchant`, if `xxx_merchant` is omitted it will default to have the catalog value.
+      - If both fields `xxx` and `xxx_customer` are omitted from the request, they will default to the value of `xxx_merchant`.
```

Comment:
- If both fields `xxx` and `xxx_customer` are omitted from the request, they will default to the value of `xxx_merchant`.

should be 

- If both fields `xxx` and `xxx_customer` are omitted from the request, they will default to will default to the catalog value.